src/components/Preview.jsx

```diff
@@ -49,6 +54,15 @@ class Preview extends React.Component {
     openWindowWithWorkaroundForChromeClosingBug(url);
   }
 
+  _handleRefreshClick() {
+    this.props.dispatch(setRefreshTimestamp(Date.now()));
```

Comment:
So dispatching the `setRefreshTimestamp` action is definitely the right move, but the way things are currently arranged, the `Workspace` component is the only one that is ”connected“ to the Redux store. So, you’ll want to pass a callback down the component tree from `Workspace` and then have this component just call the callback it’s given.

(Full disclosure: I’m pretty strongly considering changing the convention above and using many more connected components, probably in line with the [Presentational and Container Component pattern](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0), but I think it’s a good idea to stick with the existing convention until it’s officially changed).